hack/trusted-tasks-update.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+# Resolves all tasks from the `/task` directory of this repository and any Task
+# bundles in the `$QUAY_NAMESPACE` quay.io namespace (defaults to
+# `redhat-appstudio-tekton-catalog`) and runs `ec track bundle` with those git
+# and image references in order to catalogue the latest versions in the trusted
+# tasks list in `$INPUT_IMAGE` written to `$OUTPUT_IMAGE` (both default to
+# `quay.io/redhat-appstudio-tekton-catalog/data-acceptable-bundles:latest`).
+#
+# By default both git and image refrences are collected, what is collected can
+# be controlled by `$COLLECT` which can be set to `git` or `oci`
+#
+# Parameters via environment variables:
+# COLLECT        - can be set to `oci` or `git` (defaults to both), what to
+#                  resolve
+# INPUT_IMAGE    - Conftest OCI bundle containing the trusted tasks list, defaults
+#                  to:
+#                  `quay.io/redhat-appstudio-tekton-catalog/data-acceptable-bundles:latest`
+# OUTPUT_IMAGE   - Conftest OCI bundle to write the trusted task list to, defaults
+#                  to `$INPUT_IMAGE`
+# QUAY_NAMESPACE - Quay namespace to query for Task bundles (defaults to
+#                  `redhat-appstudio-tekton-catalog`)
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+COLLECT=${COLLECT:-git oci}
+mapfile -td ' ' < <(echo "${COLLECT}")
```

Comment:
I can't grok this line. What var is being set? Should it be this?
```suggestion
mapfile COLLECT -td ' ' < <(echo "${COLLECT}")
```

Reply:
Also I can't make the space delimiter work like that..

```
$ mapfile BAZ -td ' ' < <(echo "foo bar")
$ echo ${BAZ[0]}
foo bar

$ mapfile BAZ < <(printf "foo\nbar")
$ echo ${BAZ[0]}
foo
```

Reply:
I figured it out, the variable name needs to be after the parameters it seems:

```
bash -c 'mapfile -td " " BAZ < <(echo -n "foo bar"); echo "BAZ[0]: ${BAZ[0]}"; echo "BAZ[1]: ${BAZ[1]}"'
```